### ISSUE TITLE
Add Brother drivers

### ISSUE THREAD

**Issue by wschaeffer** [olbat/dockerfiles#72]
Is it possible to add Brother printer drivers? The current list in CUPS is far from complete. This driver should work: https://github.com/pdewacht/brlaser
It's is available in the Debain repository, named ```printer-driver-brlaser ```

**Comment by olbat** [comment]
Oh wait, actually `printer-driver-brlaser` should already be available in the image as it's a dependency of [printer-driver-all](https://packages.debian.org/bullseye/printer-driver-all) which is included. How did you come up to the conclusion that the driver was not installed?

**Comment by wschaeffer** [comment]
Good point, I noticed that not all available printer series are available in CUPS according to this list: https://github.com/pdewacht/brlaser In my case I miss the ```Brother HL-L2300D series``` driver.
I noticed that Bullseye ships with the latest version of brlaser: https://packages.debian.org/bullseye/printer-driver-brlaser 
Is the base docker image based on Debian buster? That would explain the difference.
